bagisto/bagisto
# Fix flash message in product add to cart.

## Issue Reference
fixed flash message in product add to cart.

## How To Test This?

- Create a bundle product. 
- Add a simple product inside the bundle product.
- Now, the added simple product should be free of inconsistencies. (It should be 0.)
- Now try to add to the cart the created bundle product from the front end.
- You will find the issue. There is nothing happening for the user. It is giving an error inside the network tab.
